.pre-commit-config.yaml

```
@@ -72,7 +72,7 @@ repos:
   #        args: ["--fix"]
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

Let's not.
```suggestion
    rev: v3.1.0
```

I'd be hesitant to switch to an unsupported alpha version. (Not sure if it is sustainable to use at all, given the comment, but for now.